config/routes.rb

```diff
@@ -1,14 +1,21 @@
 Rails.application.routes.draw do
+  get 'posts/new'
   get 'posts/index'
+  get 'comments/create'
   get 'posts/show'
   get 'users/index'
   get 'users/show'
+  get 'likes/create'
+
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   root 'users#index'
 
+  resources :comments, only: [:new, :create]
+  resources :likes, only: [:create]
```

Comment:
- [ ] Kindly wrap your resources comments and likes inside the post resources, this will make it easier to create easy-to-follow paths for likes and comments.

```suggestion
  resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: [:new, :create]
      resources :likes, only: [:create]
    end
```